### ISSUE TITLE
WebHelper - Preview

### ISSUE THREAD

**Issue by caltrop-3** [geany/geany-plugins#764]
I got nothing. Big Blank White Nothing.

![geany - webhelper](https://user-images.githubusercontent.com/31071746/43840768-7e6a20d2-9aef-11e8-840b-eb1fe4440dde.png)

**Comment by b4n** [comment]
WebHelper is not exactly a preview for your currently edited HTML file.
Most web development nowadays are a lot more than one HTML file, the page delivered to the client is usually generated a way or another; so it is seldom enough to simply display the page you're editing. Because of this, the WebHelper plugin is displaying a URI of your choice (here in your example, it's displaying `about:blank`), which can be any scheme supported by WebKit: `http`, `file`, etc.

It was suggested to add a feature to ease displaying the currently edited file, but there was not enough interest for anybody to implement it.
If you wanted real-time update (instead of having to save the file first), one could investigate the idea of loading a data URI that changes dynamically, or any other available mean through the WebKit API.

Anyway, basically to see you page you need to make the WebHelper URI point to `file:///home/caltrop/Web Page/LINUX.htm`.

**Comment by caltrop-3** [comment]
I tried it and... well I can just do that with a browser.
Also when WebHelper is configured for a separate window that window hogs the viewport.In other words you can not get at Geany anymore.
I find this Plug-In worthless.


    On Wednesday, August 8, 2018, 10:12:11 AM EDT, Colomban Wendling <notifications@github.com> wrote:  
 
 
WebHelper is not exactly a preview for your currently edited HTML file.
Most web development nowadays are a lot more than one HTML file, the page delivered to the client is usually generated a way or another; so it is seldom enough to simply display the page you're editing. Because of this, the WebHelper plugin is displaying a URI of your choice (here in your example, it's displaying about:blank), which can be any scheme supported by WebKit: http, file, etc.

It was suggested to add a feature to ease displaying the currently edited file, but there was not enough interest for anybody to implement it.
If you wanted real-time update (instead of having to save the file first), one could investigate the idea of loading a data URI that changes dynamically, or any other available mean through the WebKit API.

Anyway, basically to see you page you need to make the WebHelper URI point to file:///home/caltrop/Web Page/LINUX.htm.

**Comment by codebrainz** [comment]
@caltrop-3 what doesn't it do that you want?

**Comment by elextr** [comment]
Closed because of no constructive response from the OP.
